### ISSUE TITLE
LED lightbulb efficiency

### ISSUE THREAD

**Issue by jgvanantwerp** [MatthewHeun/MCBook2021#115]
In section 7.5, it says "LED lamps are 6–10 times more efficient than incandescent lighting."
1. provide citation.
2. it's unclear what this means. It might mean that a 100 W equivalent LED lightbulb might use 10 to 16.7 W (for the same amount of light). Or we could ask what the efficiency of a 100 W bulb is. Given chapters 5 and 6 (intensity vs. efficiency) students might expect lightbulb efficiency  to be in W out divided by W in. There is a power value in light but it is rather small. Even at 6x to 10x LED bulbs would be rather poor on this metric.

TLDR: is there a better way to state the statistic?
